Automattic/wp-calypso
# Activity Log / Rewind: Clone Site Interface

Due to the changes in https://github.com/Automattic/wp-calypso/pull/24951, the original PR for the clone site interface (https://github.com/Automattic/wp-calypso/pull/24741) needed to be reverted. This is a copy of that PR with the new data layer functionality installed.

---

Alternate Restores for Rewind! Feature flagged to be available in all environments except production for testing.

**Testing Instructions**

You are welcome to spin up a local development environment on this branch, or click the calypso.live link below.
Select a site which is rewind enabled with an active state (rewind buttons visible in the activity log).
Navigate to Settings > General and scroll to the bottom. Under Site Tools, you should have a "Clone" option. Click that button to enter the flow.
In step 2, provide your new site's title and public URL. Your public URL must be a working WordPress install, and must be prefixed with http:// or https://.
In step 3, provide the server credentials which correspond to the public URL entered in step 2. They must be active/working credentials. Make sure to add the server path that points to the install in the advanced settings.
Continue through the flow and ensure the restore runs and completes, either successfully or not. Restore failures are not a failure of this PR, but please do report those as separate issues in the jetpack-backups repository if necessary.
